initial content replacements, structural edits to follow soon

urcades opened the pull request.

urcades committed:
initial content replacements, structural edits to follow soon

matildepark commented on content/understanding-urbit/urbitos1-osn.md:
```diff
@@ -0,0 +1,63 @@
++++
+title = "Urbit OS1 –> OSN"
```
OS¹ -> OSⁿ? Hehe.

urcades replied:
Yeah, I'd personally prefer this styling tbh

matildepark replied:
Up to you

urcades committed:
list.html updated with actual structure, proper link formatting to follow

urcades committed:
UU Copy mostly finalized (needs a link audit) + basic bottom CTA on all pages

All of the copy elements of the site are up to parity with the author documents  — the only remaining thing to do is check that each link links appropriately and to the correct/intended location, as I had to populate each link myself based on the context of the writing.

Secondly, I've updated the CTA previously only located in "Project History, and it now lives on every UU page. I'd like to visually update this post-release, so consider this version an MVP.

The remaining task is to get the imagery up, bar any issues that come up in review.

urcades commented on content/understanding-urbit/urbit-id.md:
```diff
@@ -9,27 +9,27 @@ hide_previous_title = "true"
 First, let’s talk about Urbit ID’s implementation and design goals, then about where we envision it going in the future.
 
-If you’re curious to see exactly how this system works, the code is of course [open source](). Including the source for our [phonemic naming system]() and the [visual representations](). There’s even a [Figma plugin]() for using them. There’s also an interface for interacting with Urbit ID whose source is [open too]().
+If you’re curious to see exactly how this system works, the code is of course [open source](https://github.com/urbit/urbit). Including the source for our [phonemic naming system]() and the [visual representations](https://github.com/urbit/sigil-js). There’s even a [Figma plugin](https://github.com/urbit/sigil-figma-plugin) for using them. There’s also an interface for interacting with Urbit ID wh ...
```
The link for the source for the "phonemic naming system" needs to be inserted here, I have no idea what this is referring to.

matildepark replied:
Could be [urbit-ob](https://github.com/urbit/urbit-ob/blob/master/src/internal/co.js). Or the appropriate section from hoon.hoon.

urcades replied:
I'll use what you've linked!

urcades replied:
TYVM

urcades reviewed (commented):
Howdy @matildepark + @eriknewton + @galenwp , please give this update a general run-down.

The main thing to make sure is correct (i think) is the linking, since links were not provided in the source docs.

matildepark commented on content/blog/simple-durable-yours.md:
```diff
@@ -1,11 +1,8 @@
+date = 2019-10-17
 description = "The core ideas that Urbit is built around"
-weight = 1
 [extra]
```
I feel like these could use a touch up for their description and the author metadata. They look out of place in the blog directory.

![image](https://user-images.githubusercontent.com/20846414/71134446-5c12c380-21cc-11ea-9df9-b3f8a586e29c.png)

urcades replied:
Good catch

matildepark commented on content/understanding-urbit/history-people.md:
```diff
@@ -0,0 +1,125 @@
+
+Urbit.live started selling Urbit IDs very early once that became possible, and also plans to offer services within the Urbit ecosystem. 
+
+The community of contributors and core developers has been steadily contributing to the code base since before Tlon existed, and they’re still going strong. They can primarily be found on Urbit itself, in `~dopzod/urbit-help`, the [urbit-dev mailing list](https://urbit.us11.list-manage.com/subscribe/post?u=972a03db9e0c6c25bb58de8c8&id=be143888d2) and on [GitHub](https://github.com/urbit/urbit/iss ...
```
urbit-dev is [here](https://groups.google.com/a/urbit.org/forum/#!forum/dev).

GitHub should link to just urbit/urbit.

matildepark replied:
The urbit-dev link is still pointing to Mailchimp, not to the Google group.

matildepark commented on content/understanding-urbit/history-people.md:
```diff
@@ -0,0 +1,125 @@
+
+If you’re interested in homesteading a community when OS 1 comes out, [apply here](community/community-grants).
+
+Or if you’re curious to try hoon school, [let us know](/community/hoonschool).
```
capitalise

matildepark commented on content/understanding-urbit/history-people.md:
```diff
@@ -0,0 +1,125 @@
+
+Keep in touch while we make build this vision into reality by [joining our mailing list](https://urbit.us11.list-manage.com/subscribe/post?u=972a03db9e0c6c25bb58de8c8&id=be143888d2) or following us on [twitter/urbit](https://twitter.com/urbit) or [github/urbit](https://github.com/urbit). 
+
+If you’re somewhat technical and you’d like to help and contribute, [install urbit](/using/install) and consider [applying for a grant](https://grants.urbit.org/s).
```
capitalise Urbit

matildepark commented on content/understanding-urbit/history-people.md:
```diff
@@ -0,0 +1,125 @@
+
+Building Urbit is about building a future where technology doesn’t control us. Instead, we look forward to a future where technology is simpler, more reliable and less invasive. 
+
+Keep in touch while we make build this vision into reality by [joining our mailing list](https://urbit.us11.list-manage.com/subscribe/post?u=972a03db9e0c6c25bb58de8c8&id=be143888d2) or following us on [twitter/urbit](https://twitter.com/urbit) or [github/urbit](https://github.com/urbit). 
```
I'm unsure if "twitter/urbit" is the intentional link name.

eriknewton replied:
let's just call it "twitter"

matildepark commented:
If we're introducing a new roadmap, then we should remove the link to ~2019.10's Roadmap on templates/index.html, too.

matildepark commented on templates/understanding-urbit/list.html:
```diff
@@ -32,87 +32,79 @@
+  <p class="full mt0 mb4 mb4-xl lh-copy">Urbit ID is an identity and authentication system specifically designed to work with Urbit OS. When you boot or log in to Urbit OS, you use your Urbit ID. </p>
+  <p class="full mt0 mb4 mb4-xl lh-copy">Your Urbit ID is a short, memorable name (like </code>~ravmel-ropdyl</code>) that’s a username, network address and crypto wallet all in one. It’s registered on a blockchain, you own it with a key and no one can take it away from you. Urbit IDs are cryptographic property. </p>
+  <p class="full mt0 mb4 mb4-xl lh-copy">Urbit IDs aren’t money, but they are scarce, so each one costs something. This means that when you meet a stranger on the Urbit network, they have some skin in the game and are less likely to be a bot or a spammer.</p>
+  <p class="full mt0 mb4 mb4-xl lh-copy">The Urbit ID system is live and deployed, and we don’t plan to change it all that much. It works well. If you’re curious to dive further into the details, jump ahead to <a class="bb" href="/understanding-urbit/urbit-id">this post</a></p>
```
This needs a period at the end of the paragraph, it's dangling.

matildepark commented on templates/understanding-urbit/list.html:
```diff
@@ -32,87 +32,79 @@
+  </div>
+  </form>
+  <p class="full mt3 mb3 mb3-xl lh-copy">You can also follow us on <a class="bb" href="https://twitter.com/urbit">Twitter</a> or <a class="bb" href="https://github.com/urbit">Github</a>.</p>
+  <p class="full mt0 mb3 mb4-xl lh-copy">Or, preferably, on Urbit itself in <code>`~dopzod/urbit-help`</code></p>
```
If you're using a code element then the backticks should be removed.

matildepark commented on templates/understanding-urbit/list.html:
```diff
@@ -32,87 +32,79 @@
-    Urbit is a large, ambitious project. But at the core it’s simple, durable and yours.
-  </p>
+<section class="c4-10-lg">
+  <h1 id="introduction" class="mb4 full mt4 ">What Urbit Is</h1>
```
Just personal taste, but "What Urbit Is" looks weird and almost internal on the actual deploy. "Introduction" is more natural, and all the links say that as well.

eriknewton replied:
Agreed.

matildepark commented on content/understanding-urbit/history-people.md:
```diff
@@ -0,0 +1,125 @@
+
+At the beginning, Urbit was just a few people with the right combination of imagination and discipline to try to rebuild computing.
+
+2002
```
This timeline [doesn't have the line breaks](https://deploy-preview-329--urbit-org.netlify.com/understanding-urbit/history-people/) in the deploy. You'll need to add another line break.

urcades committed:
Pushing new images, still a few left

Pushing the majority of imagery, still struggling over a few of the new additions to the spec in roadmap and history + people

urcades committed:
More images, pushing Matilde review updates

urcades committed:
more small PR tweaks per @matildepark

matildepark commented on content/understanding-urbit/history-people.md:
```diff
@@ -0,0 +1,136 @@
+
+For those interested in contributing there are some simple instructions on [GitHub](https://github.com/urbit/urbit/blob/master/CONTRIBUTING.md#contributing-to-urbit), but we also run a [grants program](https://grants.urbit.org/) where we reward contributors with Urbit ID address space. Grants aren’t solely for pre-determined projects, either. If you have something you’d like to propose, we’d love ...
+
+We also run a bi-monthly course called ‘Hoon School’ to teach people about the system. Hoon School is designed for anyone with the most basic technical skills — you don’t have to be an expert programmer. If you’re interested, you should [apply](community/hoonschool). We accept everyone we can and only close the class when it gets too big.
```
should be `/community/hoonschool`

matildepark commented on content/understanding-urbit/history-people.md:
```diff
@@ -0,0 +1,136 @@
+
+If you’re somewhat technical and you’d like to help and contribute, [install Urbit](/using/install) and consider [applying for a grant](https://grants.urbit.org/s).
+
+If you’re interested in homesteading a community when OS 1 comes out, [apply here](community/community-grants).
```
should be `/community/community-grants`

matildepark commented on content/understanding-urbit/roadmap.md:
```diff
@@ -0,0 +1,115 @@
+
+If OS 0 is for explorers, OS 1 is for tinkerers. More like car camping than backpacking.
+
+Tlon is also planning to provide free hosting to a limited number of groups who want to homestead the system. (Although you can always host Urbit yourself, of course.) If you’re interested in trying OS 1 with your group of friends, [let us know](community/community-grants). 
```
`/community/community-grants`

matildepark commented on templates/understanding-urbit/list.html:
```diff
@@ -32,87 +32,79 @@
+  <p class="full mt0 mb4 mb4-xl lh-copy">If you’re curious to dive even deeper into Urbit OS, feel free to <a class="bb" href="/understanding-urbit/urbit-os">skip ahead</a>.</p>
+  <img class="full mb4" src="https://media.urbit.org/site/understanding-urbit/project-history/intro-id.svg"/>
+  <p class="full mt0 mb4 mb4-xl lh-copy">Urbit ID is an identity and authentication system specifically designed to work with Urbit OS. When you boot or log in to Urbit OS, you use your Urbit ID. </p>
+  <p class="full mt0 mb4 mb4-xl lh-copy">Your Urbit ID is a short, memorable name (like </code>~ravmel-ropdyl</code>) that’s a username, network address and crypto wallet all in one. It’s registered on a blockchain, you own it with a key and no one can take it away from you. Urbit IDs are cryptographic property. </p>
```
`</code>~ravmel-ropdyl</code>` should be `<code>~ravmel-ropdyl</code>`

urcades committed:
final image updates + link fixes, please clap/review

urcades committed:
tiny updates/fixes to "understanding-urbit/people-history"

urcades committed:
footer pagination linkfix

urcades committed:
footer cta fixes across all pages

urcades committed:
<code> enclosure fix

urcades committed:
svg margin/width fixes across all pages

urcades committed:
last link fixes per @matildepark  review

galenwp committed:
a few tweaks

galenwp commented:
The only things I see missing:

- Homepage content from the Figma mocks
- 1px borders on the images. I like how we were consistent about this before.

matildepark commented:
I'm working on that feedback — I noticed Roadmap and OS1 -> N are the same post, but a lot of the content links to OS1 -> N, so I'm gonna rectify that and add an alias in the post, too.

matildepark committed:
add borders to all imgs

matildepark committed:
fixing the links to point to interface

matildepark committed:
edit index.html

galenwp commented:
Good catch, thanks.

matildepark commented:
Ready for another look @galenwp.

galenwp committed:
copy edits, fixes

galenwp commented:
Edited the copy, changed the homepage a bit, take a look

matildepark commented:
![image](https://user-images.githubusercontent.com/20846414/71203531-1f96a480-226c-11ea-97e9-5313b12e73ab.png)

I don't know about the italics and I really don't think we need to link to UU. It's directly below it.

galenwp commented:
Reading that intro copy, I found it really confusing to see 'Install Urbit OS' above UU. For people who have never seen Urbit before, I want to make sure they head to UU before trying to install.

matildepark commented:
There are very few situations in which a fresh user won't be able to see the words "Understanding Urbit", like, directly below that installation link, and it looks redundant and sort of odd (why is it green? What is that trying to signify?).

Some of our users are going to come knowing what Urbit is and wanting the install link. An increasing number, really — they have to come back for updated binaries. The original one had "Introduction" in black, install and docs below it for alternative streams into the site if you know what you want — very natural. 

And if we want people to return to this site — to tap our content streams — we need to stop so aggressively formulating an introductory funnel for everyday people to read it and bounce.

I'd even go so far as to say that "Introduction / Install / Documentation" link titles are optimal, cleanest.

But I know @eriknewton said this is iterative and meant to go out ASAP, so I can see very little room to argue much further here. You just said to take a look.

galenwp commented:
Sending new visitors to 'install urbit' is going to be confusing. Anyway, this is a layout issue. I think those links are a relic and can go. Ed has a way to reorganize the text into a three-up that he's working on. I'll push something that should work for the time being.

galenwp committed:
simplify the index
